.github/workflows/blogs-indexing.yaml

```diff
@@ -0,0 +1,46 @@
+name: Indexing blogs
+on:
+  push:
+    branches: ['search-blogs-ci-indexing']
+    paths:
+      - 'content/blog/**'
+jobs:
+  build-and-indexing-blogs:
+    name: Build and indexing blogs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node.js and use yarn
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+      - name: create env file
+        run: |
+          touch .env
+          echo GATSBY_ALGOLIA_APP_ID=${{ secrets.GATSBY_ALGOLIA_APP_ID }} >> .env
+          echo ALGOLIA_ADMIN_KEY=${{ secrets.ALGOLIA_ADMIN_KEY }} >> .env
+          echo GATSBY_ALGOLIA_INDEX_NAME=${{ secrets.GATSBY_ALGOLIA_INDEX_NAME }} >> .env
```

Comment:
Getting secrets for Algolia from Github Action Secret.

Reply:
@shcheklein Can I get access to setting these secrets so that I can check/verify the data are getting indexed properly to Algolia?

Reply:
Yeah, I had tried and it wasn't working. So, I changed to this way. But, I found later that the value wasn't set properly. So, yeah setting `env` should also work.